README.md

```diff
@@ -44,65 +45,106 @@ Usage instructions are available in the bootstrap [README](./0-bootstrap/README.
 
 ### [1. org](./1-org/)
 
-The purpose of this stage is to set up top level folders used to house projects which contain shared resources such as monitoring, networking, org level logging and also to set baseline security settings through organizational policy.
+The purpose of this stage is to set up the common folder used to house projects which contain shared resources such as DNS Hub, Interconnect, SCC Notification, org level secrets and org level logging.
 This will create the following folder & project structure:
 
 ```
 example-organization
 └── common
-    ├── logs
-    │   ├── org-audit-logs
-    │   └── org-billing-logs
-    ├── monitoring
-    │   ├── org-monitoring-nonprod
-    │   └── org-monitoring-prod
-    └── networking
-        ├── org-shared-vpc-nonprod
-        └── org-shared-vpc-prod
+    ├── prj-p-org-audit-logs
+    ├── prj-p-org-billing-logs
+    ├── prj-p-org-dns-hub
+    ├── prj-p-org-interconnect
+    ├── prj-p-org-scc
+    └── prj-p-org-secrets
 ```
 
 #### Logs
 
-Under the logs folder, two projects are created. One for organization wide audit logs and another for billing logs.
+Under the common folder, two projects are created. One for organization wide audit logs and another for billing logs.
 In both cases the logs are collected into BigQuery datasets which can then be used general querying, dashboarding & reporting.
 For the various audit log types being captured in BigQuery, this is mirroring the standard [retention periods](https://cloud.google.com/logging/quotas#logs_retention_periods) for Cloud Logging, these settings can be adjusted according to your requirements.
```

Comment:
For this, the logs are no longer following standard retention for audit logs - so it might be best to say, they are retained for 30 days

Reply:
Should we also mention that the logs are being sent to pub/sub and GCS?

Reply:
Yes!